config/puma.rb

```diff
@@ -2,6 +2,8 @@
 threads_count = Integer(ENV['RAILS_MAX_THREADS'] || 5)
 threads threads_count, threads_count
 
+worker_timeout 600
```

Comment:
:-1: this should at a minimum be configurable if we want to add it at all

Reply:
Ah good catch; I did this for debugging and forgot to remove it.